InfluxDB.Client.AspNetCore/InfluxDB.Client.AspNetCore.csproj

```
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
```

Please add also support for `netstandard2.1` and set `LangVersion` to `8`:

```suggestion
  <PropertyGroup>
    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
    <LangVersion>8</LangVersion>
  </PropertyGroup>
```